src/router/routes/job.js

```diff
@@ -166,25 +189,13 @@ const handleDeploymentJob = async (job, clean = true) => {
 
     switch (job.deploymentPlatform) {
       // Enterprise deployment refer to Airwatch:
-
       case 'enterprise': {
         deployedAppPath = await deployAirWatch(job.originalFileName, job.platform, job.awOrgID, job.awFileName); // Pass in extra parameters for AW
         break;
       }
       // Public deployment refer to Apple or Google Store, depends on application type:
       case 'public': {
-        switch (job.platform) {
-          case 'ios': {
-            deployedAppPath = await deployAppStore(job.originalFileName);
-            break;
-          }
-          case 'android': {
-            deployedAppPath = await deployGoogle(job.originalFileName);
-            break;
-          }
-          default:
-            throw new Error('Unsupported application type');
-        }
+        deployedAppPath = await switchDeploymentPlatform(job.platform, job.originalFileName);
```

Comment:
I don't think you want to use the name like this `switchDeploymentPlatform`. Better to use something like `deploymentPath` (or selectDeploymentPath).